lib/movable_ink/aws/lambda.rb

```diff
@@ -0,0 +1,39 @@
+require 'aws-sdk-lambda'
+
+module MovableInk
+  class AWS
+    module Lambda
+      def lambda(region: 'us-east-1')
+        @lambda_client ||= {}
+        @lambda_client[region] ||= Aws::Lambda::Client.new(region: region)
+      end
+
+      def disable_autoscaling_lambda
+        function_name = 'autoscaling-scheduled-capacity-production-cleanup'
```

Comment:
Please consider making `function_name` an argument for `disable_autoscaling_lambda()` so we don't expose MI infrastructure into public gem and don't need newer version if the name ever changes.